mass_mailing_custom_unsubscribe/static/src/js/unsubscribe.js

```diff
@@ -3,35 +3,37 @@
 
 /* This JS module replaces core AJAX submission because it is impossible
  * to extend it as it is currently designed. It is almost a copy+paste from
- * upstream, to allow easier version/patch updates, so linter is disabled. */
+ * upstream, to allow easier version/patch updates, so linter is disabled
+ * and prettier is ignore. */
 /* eslint-disable */
+// prettier-ignore
```

Comment:
This might be a little nitpicking, but could you add this little part in a commit _before_ passing prettier to it? Otherwise we have a commit passing prettier to this file, and another commit undoing it, and generates a big diff that makes it difficult to see what has really changed here downstream.